examples/homepage/.env.example

```
@@ -6,6 +6,10 @@ NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=
 NEXT_PUBLIC_DELEGATOR_URL=
 # Network Configuration -- main | test | solo
 NEXT_PUBLIC_NETWORK_TYPE=
+# Transak onramp -- staging key from dashboard.transak.com (leave empty to hide the Buy button)
+NEXT_PUBLIC_TRANSAK_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_TRANSAK_API_KEY key should go before the NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID key

(UnorderedKey)